### PR TITLE
Deprecate testing/environment in favor of elastic-package stack

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

The PR deprecates the `testing/environment` setup in favor of the `elastic-package stack` command.

## How to test this PR locally

Follow the new guide for booting up a local stack. Try to modify the integration and reload these changes in Kibana.

## Related issues

- Resolves https://github.com/elastic/integrations/issues/404

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,14 +88,16 @@ feel free to review the script's [README](https://github.com/elastic/integration
...
+3. Use the `elastic-package stack up` command to boot up required dependencies:
     1. Elasticseach instance:
         * Kibana's dependency
     2. Kibana instance:
         * used to migrate dashboards, if not available, you can skip the generation (`SKIP_KIBANA=true`)
 
-    _Hint_. There is dockerized environment in beats (`cd testing/environments`). Boot it up with the following command:
-    `docker-compose -f snapshot.yml up --force-recreate`.
+    ~~_Hint_. There is dockerized environment in beats (`cd testing/environments`). Boot it up with the following command:
+    `docker-compose -f snapshot.yml up --force-recreate`.~~ (deprecated)
```
Comment (ycombinator) [inline CONTRIBUTING.md:98]: Any reason we should leave this in here, marked as deprecated? I'm wondering if we can just remove this method altogether now?

**Review by ycombinator — COMMENTED** [review]
My main feedback is about the documentation changes. 
- Rather than striking out the old method and marking it as deprecated, I would suggest simply showing the new method of achieving the same thing. I think this is clearer for the user. 
- In many places in the `CONTRIBUTING.md` you have referred to the cheat sheet for `elastic-package`. I think this is great but I would also provide the exact replacement command(s) to run with `elastic-package`. From what I can tell `mage Reload` was rebuilding all packages and then rebuilding+starting the package registry container, so I would simply provide the equivalent `elastic-package` commands inline in the `CONTRIBUTING.md` in place of `mage Reload`. This will save users an extra hop to the cheat sheet.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -264,21 +266,29 @@ what's been already fixed, as the script has overridden part of it).
...
+
    ```bash
-   $ cd testing/environments
-   $ docker-compose -f snapshot.yml up
+   $ elastic-package stack up -d -v
    ```
 
-   The command will boot up a docker cluster with Elasticsearch, Kibana and Package Registry. After every time you
-   rebuild and reload packages (`mage Reload`), all adjustments in packages will be propagated to the registry.
+   ~~The command will boot up a docker cluster with Elasticsearch, Kibana and Package Registry. After every time you
```
Comment (ycombinator) [inline CONTRIBUTING.md:286]: I would remove the strike out presentation and simply rewrite this section with:
- what does this command do, and
- what are the `elastic-package` commands to run for reloading packages

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -381,9 +391,12 @@ on the business or technical requirements for the entire platform (Elastic Packa
 
 #### Development
 
-1. When you're developing integrations and you'd like to propagate your changes to the package registry,
-   use `mage Reload` to rebuild and reload the package registry.
-   
+1. ~~When you're developing integrations and you'd like to propagate your changes to the package registry,
```
Comment (ycombinator) [inline CONTRIBUTING.md:394]: I think it's helpful to keep the introductory sentence ("when you're developing..."). Just remove all mentions of `mage Reload` and replace them with the equivalent `elastic-stack` commands.

File: testing/environments/README.md [hunk 4]
```diff
@@ -1,26 +1,55 @@
-Before using the Package Registry, remember to `mage build` the project to prepare the volume with packages
-(`public` directory).
+## Deprecation notice
 
-Refresh docker images:
+Docker Compose files have been deprecated in favor of the `elastic-package stack` command. The tool can be found in:
+https://github.com/elastic/elastic-package . With the current building procedure, you will also find the correct binary
+in the `build` directory.
```
Comment (ycombinator) [inline testing/environments/README.md:5]: Personally I don't think this deprecation notice is necessary but I'm okay with keeping it here if you think it's worth it. If we keep it, at what point would you say its okay to remove this altogether?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mtojek) [reply CONTRIBUTING.md]: You convinced me to remove the deprecation notice. Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mtojek) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (mtojek) [reply CONTRIBUTING.md]: Thanks, fixed.

File: testing/environments/README.md (hunk 4 above)
Comment (mtojek) [reply testing/environments/README.md]: I removed the deprecation noticed.

**Comment by mtojek** [comment]
> * Rather than striking out the old method and marking it as deprecated, I would suggest simply showing the new method of achieving the same thing. I think this is clearer for the user.

I've adjusted CONTRIBUTING guide.

> * In many places in the `CONTRIBUTING.md` you have referred to the cheat sheet for `elastic-package`. I think this is great but I would also provide the exact replacement command(s) to run with `elastic-package`. From what I can tell `mage Reload` was rebuilding all packages and then rebuilding+starting the package registry container, so I would simply provide the equivalent `elastic-package` commands inline in the `CONTRIBUTING.md` in place of `mage Reload`. This will save users an extra hop to the cheat sheet.

Actually, that was goal to ask people to do this extra hop, so we don't have commands written in multiple places. It would make easier maintenance, but I'm fine to keep them in both places :)

File: testing/environments/README.md [hunk 5]
```diff
@@ -1,26 +1,49 @@
...
+
+`elastic-package stack up -d -v`
+
+Take down the stack:
+
+`elastic-package stack down -v`
 
-... or with Elastic Agent:
+Advanced: if you need to modify the internal Docker compose definition, edit files in `~/.elastic-package/stack`, but
+keep in mind that these files shouldn't be modified and your changes will be reverted once you update the `elastic-package`:
```
Comment (ycombinator) [inline testing/environments/README.md:18]: 👍

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**